module-services/service-desktop/WorkerDesktop.cpp

```diff
@@ -76,11 +75,11 @@ void WorkerDesktop::closeWorker(void)
 
     this->close();
 
+    cpuSentinel->ReleaseMinimumFrequency();
     auto sentinelRemoveMsg = std::make_shared<sys::SentinelRemovalMessage>("WorkerDesktop");
-    ownerService->bus.sendUnicast(sentinelRemoveMsg, service::name::system_manager);
+    ownerService->bus.sendUnicastSync(sentinelRemoveMsg, service::name::system_manager, 1000);
```

Comment:
we can use async -> await here and either log if not closed in 1sec or try to sync somewhere later if you want ( it seems ok to me though, this code wont' be much prettier ( just looking 5 meaningful lines of code above proves my point here :))

Reply:
It seems to me that we can safely use sendUnicastSync here - handling this message is very fast.